### PR TITLE
feat: add helm readme generation

### PR DESCRIPTION
**Pull request by galexrt**
Resolves KSD-189

Signed-off-by: Alexander Trost <galexrt@googlemail.com>

### CODE REVIEW

**Comment by galexrt** [comment]
@zalsader as you said the `make helm` overrides the comments I added.

**Comment by galexrt** [comment]
@zalsader as you said, the `make helm` overrides the comments I added. Should we ignore this for now and merge it, so we have at least the helm chart docs auto-generation already?

**Comment by zalsader** [comment]
> @zalsader as you said, the `make helm` overrides the comments I added. Should we ignore this for now and merge it, so we have at least the helm chart docs auto-generation already?

I have an idea on how to fix this. Mind if I take over the PR??

**Review by zalsader — APPROVED** [review]

File: charts/koor-operator/README.gotmpl.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+The Ceph Operator helm chart will install the basic components necessary to create a storage platform for your Kubernetes cluster.
+
+1. Add the Koor Helm repo
+2. Install the Helm chart
+3. [Create a Koor Storage cluster](https://docs.koor.tech/v1.11/Getting-Started/quickstart/#create-a-ceph-cluster).
+
+The `helm install` command deploys the Koor Operator on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation. It is recommended that the Koor Operator be installed into the `koor-operator` namespace (you will install your clusters into separate namespaces).
+
+```console
+helm repo add koor-release https://charts.koor.tech/release
```
Comment (zalsader) [inline charts/koor-operator/README.gotmpl.md:30]: This chart is not hosted here, it's hosted on github:
```suggestion
helm repo add koor-operator https://koor-tech.github.io/koor-operator
```

File: charts/koor-operator/README.gotmpl.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+helm install --create-namespace --namespace koor-operator koor-operator koor-release/koor-operator -f values.yaml
```
Comment (zalsader) [inline charts/koor-operator/README.gotmpl.md:31]:
```suggestion
helm install --create-namespace --namespace koor-operator koor-operator koor-operator/koor-operator -f values.yaml
```

**Review by zalsader — APPROVED** [review]

**Comment by galexrt** [comment]
Thanks for the help with helmify @zalsader!

**Pull request merged**